Fe profile listings page

quddd opened the pull request:
-----------------------------------------------------Shows a list of profiles----------------------------------------------
<img width="1422" alt="Screen Shot 2021-01-20 at 9 28 53 AM" src="https://user-images.githubusercontent.com/24212950/105189011-84bd6900-5b02-11eb-86fc-8778f617a73f.png">
---------------------------Responsive-----------------------------------------------------------------------------------
<img width="653" alt="Screen Shot 2021-01-20 at 9 28 32 AM" src="https://user-images.githubusercontent.com/24212950/105189076-969f0c00-5b02-11eb-8c2e-3398d327274f.png">

-------------------------------------- Only Show Sitters -----------------------------------------------------------------
<img width="1422" alt="Screen Shot 2021-01-20 at 9 29 14 AM" src="https://user-images.githubusercontent.com/24212950/105189226-bdf5d900-5b02-11eb-9a81-6703ad514edc.png">

quddd committed:
change getProfileLists to public

quddd committed:
profileListings page

fetch a list of profile and render

quddd committed:
return only Profiles that are sitters

dproc96 reviewed (commented):
This looks good except for a couple of minor nitpicks I noticed:
- In mobile view, the profile cards aren't centered, they're off to the left
- A profile card with a description is a different size than a profile card without one
- In profile listings, the user can see themselves if they are a sitter
- The spacing looks off in some of your files, be sure to run prettier before committing

dproc96 commented on client/src/pages/ProfileListings.js:
```diff
@@ -0,0 +1,154 @@
+    if(sitters.data){ // set profile data to display
+        content = 
+        sitters.data.map((sitter, key) => 
+            <ProfileCard 
```
Be sure to add a key attribute to this

quddd committed:
make search bar and date pickers seperate component

quddd committed:
FIX: make all profilecards same height

dproc96 reviewed (approved):
This looks good but just make two minor changes before merging:
1. Run Prettier on profileCard.js, searchAndFilter.js, and ProfileListings.js
2. Rename profileCard.js and searchAndFilter.js to capitalize the first letter since they're components

quddd committed:
prettier code formatting and renaming components

Renamed components to start with uppercase letter

-use prettier to format code.

quddd committed:
fixed text overflow

quddd committed:
Update profileCard.js

quddd commented:
****Fixed** Text overflow**
![Uploading Screen Shot 2021-01-21 at 12.05.42 PM.png…]()

quddd committed:
Remove the user profile from the sitters list

quddd committed:
Merge branch 'dev' into FE-Profile-Listings-page

quddd committed:
Revert "Merge branch 'dev' into FE-Profile-Listings-page"

This reverts commit e2650cb39ecbdabd0d3f86d86d676e2292b21f7f, reversing
changes made to e7494c98799e6e9f7819edbe0e502eeb30f33fae.

quddd committed:
Revert "Revert "Merge branch 'dev' into FE-Profile-Listings-page""

This reverts commit 7e40390b24f863acd123eed77d3ed65990370ea3.

quddd committed:
Revert "Merge branch 'dev' into FE-Profile-Listings-page"

This reverts commit e2650cb39ecbdabd0d3f86d86d676e2292b21f7f, reversing
changes made to e7494c98799e6e9f7819edbe0e502eeb30f33fae.
